request_handlers_test.cc

```diff
@@ -106,3 +116,172 @@ TEST_F(RequestHandlersTest, OutputWithHeader) {
   EXPECT_STREQ(test_response_buffer, response_buffer)
     << "response_buffer should be the same string as test_response_buffer";
 }
+
+TEST_F(RequestHandlersTest, HTMLTest) {
+  const char request_buffer[] = "GET /static1/index.html HTTP/1.1\r\n"
+                          "Host: localhost:8080\r\n"
+                          "Accept-Encoding: gzip, deflate, compress\r\n"
+                          "Accept: */*\r\n"
+                          "User-Agent: HTTPie/0.8.0\r\n\r\n";
+  const size_t request_buffer_size = strlen(request_buffer);
+
+  char* response_buffer = nullptr;
+  size_t response_buffer_size = 0;
+
+  const char expected_response_buffer[] =
+    "HTTP/1.1 200 OK\r\nContent-Type: text/html\r\n\r\n<!DOCTYPE html>\n<html>\n"
```

Comment:
Let's format this similar to what we did for the 404 handler in request_handler.cc? Or similarly, like in the RouteDoesNotExist test below.